expect/_to_be_close_to.ts

```diff
@@ -0,0 +1,35 @@
+// Copyright 2018-2023 the Deno authors. All rights reserved. MIT license.
+
+import { MatcherContext, MatchResult } from "./_types.ts";
+import { AssertionError } from "../assert/assertion_error.ts";
+
+// TODO(kt3k): tolerance handling is wrong
```

Comment:
Is there a test that covers this tolerance?

Reply:
Oops, this has been already resolved in https://github.com/denoland/deno_std/pull/3814/commits/357ecb3f2cd0f2ab9a2748cc7793c6f9f8f3c1ef